src/Leaflet.Editable.js

```diff
@@ -164,6 +165,16 @@ L.Editable = L.Class.extend({
         return marker;
     },
 
+    startRectangle: function(latlng, options) {
+        var corner = latlng || new L.LatLng(0, 0);
```

Comment:
Let's move like this for now, but then I think I need to refactor this pattern so we can call `startRectangle` while actually creating the rectangle (and other features) only at first map click. So we don't need the isZero trick, and same for Marker which is put on the middle of the map by convention, but that's quite ugly :/